<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>880</width>
    <height>472</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Neural Simulation</string>
  </property>
  <widget class="QWidget" name="centralWidget">
   <widget class="QCustomPlot" name="customPlot" native="true">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>10</y>
      <width>661</width>
      <height>431</height>
     </rect>
    </property>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menuBar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>880</width>
     <height>26</height>
    </rect>
   </property>
   <widget class="QMenu" name="menuAction">
    <property name="title">
     <string>Action</string>
    </property>
    <addaction name="actionCalcul"/>
    <addaction name="actionDraw_1"/>
    <addaction name="actionDraw_2"/>
    <addaction name="actionDraw_Nullcline"/>
    <addaction name="actionDraw_All"/>
   </widget>
   <addaction name="menuAction"/>
  </widget>
  <action name="actionCalcul">
   <property name="text">
    <string>Calcul</string>
   </property>
  </action>
  <action name="actionDraw_1">
   <property name="text">
    <string>Draw 1</string>
   </property>
  </action>
  <action name="actionDraw_2">
   <property name="text">
    <string>Draw 2</string>
   </property>
  </action>
  <action name="actionDraw_Nullcline">
   <property name="text">
    <string>Draw Nullcline</string>
   </property>
  </action>
  <action name="actionDraw_All">
   <property name="text">
    <string>Draw All</string>
   </property>
  </action>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <customwidgets>
  <customwidget>
   <class>QCustomPlot</class>
   <extends>QWidget</extends>
   <header location="global">qcustomplot.h</header>
   <container>1</container>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections/>
</ui>
